tasks/install-on-Darwin.yml

```
@@ -13,13 +13,24 @@
     state: latest
     update_homebrew: yes
 
+- name: Configure Buildkite service
+  template:
+    src: "homebrew.mxcl.buildkite-agent.plist.j2"
+    dest: "{{ buildkite_agent_conf_dir[ansible_os_family] }}/homebrew.mxcl.buildkite-agent.plist"
+    owner: "{{ buildkite_agent_username }}"
+    group: staff
+    mode: "0600" # macOS requires 0400 or 0600.
+  notify:
+    - "restart-darwin-buildkite"
+  when: ansible_os_family == "Darwin" and buildkite_agent_allow_service_startup[ansible_os_family]
```

```suggestion
  when: buildkite_agent_allow_service_startup[ansible_os_family]
```

It's unnecessary to say when Darwin because we're in the Darwin-specific file.